tests/contrib/config.py

```diff
@@ -95,7 +95,7 @@
 
 MOTO_CONFIG = {
     "host": os.getenv("TEST_MOTO_HOST", "127.0.0.1"),
-    "port": int(os.getenv("TEST_MOTO_PORT", "5000")),
+    "port": int(os.getenv("TEST_MOTO_PORT", "3000")),
```

Comment:
why this change?

Reply:
Moto runs on port 3000, took me awhile to notice this when my `aiobotocore` tests were failing. on CI they set this env, but best to just change it to be the actual correct port as the fallback